src/requestor/routes/manage.py

```diff
@@ -111,7 +145,7 @@ async def create_request(
         token_username = token_claims["context"]["user"]["name"]
         logger.debug(f"Got username from token: {token_username}")
         data["username"] = token_username
-
+    data["revoke"] = False
```

Comment:
?

Reply:
I think I can remove this one. I think I've added it just in case when we're trying to fetch the key when it is absent, but replacing it with `data.get("revoke",False)` should fix the issue. Will update it accordingly.